examples/networking/router_route/main.tf

```diff
@@ -0,0 +1,9 @@
+resource "vkcs_networking_router_route" "route_to_internal_network" {
+  depends_on = [
+    vkcs_networking_router_interface.app,
+    vkcs_networking_port.port-for-subnet-tf-example
+  ]
+  router_id = vkcs_networking_router.router.id
+  destination_cidr = "192.168.199.0/24"
+  next_hop = "192.168.199.6"
```

Comment:
i have no idea where this IP came from
lets remove this example

Reply:
removed